rulebook.tex

```diff
@@ -612,26 +670,182 @@ \subsection{Machines}
-            Delivery Station (DS) & 1 per team &$t_5 =\SItextrange{20:40}{\sec}$
+            \acf{BS} & 1 per team & minimum physical time\\
+            \acf{CS} & 2 per team & $t_2 = \SItextrange{15:25}{\sec}$\\
+            \acf{RS} & 2 per team &$t_3 = \SItextrange{40:60}{\sec}$\\
+            \acf{SS} & 1 per team & minimum physical time\\
+            \acf{DS} & 1 per team &$t_5 =\SItextrange{20:40}{\sec}$
         \end{tabularx}
     }
-    \caption{MPS type, distribution and processing times}
+    \caption{\ac{MPS} type, distribution and processing times}
     \label{tab:processing-times}
 \end{table}
 
+\subsubsection{Physical Description of Machines}
+
+The stations have a rectangular base shape of \SI{0,35 x 0,7}{\metre}% chktex 29
+with a height of about \SI{1}{\metre} depending on machine type.
+A machine is movable by four wheels with \SI{0.1}{\metre} clearance.
+Both narrow sides of the trolley are closed by plexiglas and have a handle.
+All physical interfaces like conveyor belt inputs and outputs, shelves, and
+slides for additional bases on ring stations are accessible at
+\SI{89.8}{\centi\metre} height\footnote{Note however, that due to small
+  variances and unevenness in the floor there may likewise be small variances
+  in the working height of some or all parts of a station.}.
+Working space between guiding lanes is \SI{4.5}{\centi\metre}. Setup lanes and
+shelves feature approximately the same space for handling and
+adjusting. All diffuse sensors (input side) have been removed to allow
+for infrared-emitting cameras.  To simplify the delivery on the belt,
+each machine will be equipped with a narrowing cone on its input side
+(\reffig{fig:narrow-cone}).  You can order them from Festo Didactic SE
+(C. Deppe) or download the STL model file from the \ac{RCLL} site.
+\begin{figure}
+    \includegraphics[height=8cm]{narrowConeSketch.jpg}
+\caption{Narrowing cone}
+\label{fig:narrow-cone}
+\end{figure}
+% Still unclear, thus commenting out.
+% If observed from the output side of machines (where the signal unit
+% is placed) the conveyor space opens up at half of the aluminum
+% profile thus starting at \SI{0,35}{\metre}.
+%\begin{figure}[b!]
+%    \centering
+%    \includegraphics[height=7cm]{MPS2015/MPS_Layout.jpg}
+%
+%    \caption{General Machine Layout}
+%    \label{fig:MPS-Layout}
+%\end{figure}
+
+\subsubsection{Machine Positioning}
+\label{sec:machine-swapping}
+The zones of each field are named according to the following scheme:
+A team prefix (``C-'' for cyan, ``M-'' for
+magenta), ``Z'' to distinguish zone names, and a grid coordinate form the name
+of a zone.
+For example, The zone C-Z23 is a zone on the cyan primary half, which is
+the second along the positive half of the X-axis and the third along the Y-axis.
+Zones will not be physically represented or visible on the field.
+They cannot be used for any other purpose than machine positioning.
```

Comment:
I would move this to the general explanation of the Area/Playing field above, and only mention here that they are in a zone.

Reply:
Makes sense, done